po/zh_CN.po

```diff
@@ -4001,74 +4001,87 @@ msgid ""
 "Column %d of item %d is length %d inconsistent with column %d which is "
 "length %d. Only length-1 columns are recycled."
 msgstr ""
+"第%d列(子项%d)的长度为%d，与第%d列的长度%d不一致。只有length-1列被循环补齐。"
 
 #: rbindlist.c:58
 #, c-format
 msgid ""
 "Column %d ['%s'] of item %d is length 0. This (and %d other%s like it) has "
 "been filled with NA (NULL for list columns) to make each item uniform."
 msgstr ""
+"第%d列['&s'](子项%d的)的长度为0，它(以及%d其他%s类似的)被填充为NA"
+(等同与列表列的NULL值)使得每一个子项统一。"
 
 #: rbindlist.c:62
 #, c-format
 msgid ""
 "Total rows in the list is %lld which is larger than the maximum number of "
 "rows, currently %d"
 msgstr ""
+"列表的总行数为%lld，它比当前行数的最大值%d大。"
 
 #: rbindlist.c:63
 msgid "use.names=TRUE but no item of input list has any names"
 msgstr ""
+"use.names=TRUE，但是输入列表的子项均没有名字。"
 
 #: rbindlist.c:71
 #, c-format
 msgid ""
 "Failed to allocate upper bound of %lld unique column names [sum(lapply(l,"
 "ncol))]"
 msgstr ""
+"分配%lld独有列名[sum(lapply(l,ncol))]的上边界失败。"
 
 #: rbindlist.c:102
 #, c-format
 msgid "Failed to allocate nuniq=%d items working memory in rbindlist.c"
-msgstr ""
+msgstr "分配nuniq=%d子项的工作内存于rbindlist.c失败。"
 
 #: rbindlist.c:136
 #, c-format
 msgid "Failed to allocate ncol=%d items working memory in rbindlist.c"
-msgstr ""
+msgstr "分配ncol=%d子项的工作内存与rbindlist.c失败。"
 
 #: rbindlist.c:191
 msgid ""
 "Internal error: usenames==NA but fill=TRUE. usenames should have been set to "
 "TRUE earlier with warning."
 msgstr ""
+"内部错误：usenames==NA但是fill=TRUE.usenames应该在警告之前被设置为TRUE。"
 
 #: rbindlist.c:196
 msgid ""
 " use.names='check' (default from v1.12.2) emits this message and proceeds as "
 "if use.names=FALSE for  backwards compatibility. See news item 5 in v1.12.2 "
 "for options to control this message."
 msgstr ""
+"use.names='check'(默认版本为v1.12.2)发出该消息并按照use.names=FALSE进行向后"
+"兼容。有关控制此消息的选项，请参见v1.12.2中item 5的消息。"
 
 #: rbindlist.c:206
 msgid ""
 "Internal error: could not find the first column name not present in earlier "
 "item"
-msgstr ""
+msgstr "内部错误：无法发现第一个列名，未出现在之前的子项中。"
 
 #: rbindlist.c:210
 #, c-format
 msgid ""
 "Column %d ['%s'] of item %d is missing in item %d. Use fill=TRUE to fill "
 "with NA (NULL for list columns), or use.names=FALSE to ignore column names.%s"
 msgstr ""
+"第%d列(第%d项的)在第%d项中丢失，使用fill=TRUE以用NA(等同于列表列的NULL)填充该列"
+"或使用use.names=FALSE以忽视名为%s的列。"
```

Comment:
Yes, close, I would say more precisely, it's filled with the value of `extra`; if `use.names=TRUE`, `extra` is just `""`, otherwise it's `" use.names='check' (default from v1.12.2) emits this message and proceeds as if use.names=FALSE for  backwards compatibility. See news item 5 in v1.12.2 for options to control this message."`.

In turn, this message was translated a few lines earlier, so the `%s` at the end will be filled with:

```
"use.names='check'(默认版本为v1.12.2)发出该消息并按照use.names=FALSE进行向后"
"兼容。有关控制此消息的选项，请参见v1.12.2中item 5的消息。"
```

Reply:
Got it. So this will combine the previous message.